packages/@uppy/companion/src/standalone/index.js

```diff
@@ -178,7 +178,11 @@ app.use((req, res, next) => {
 if (app.get('env') === 'production') {
   // @ts-ignore
   app.use((err, req, res, next) => {
-    console.error('\x1b[31m', req.id, err, '\x1b[0m')
+    // if the error is as a result of bad request from client, no need
+    //  to flood production logs with this.
+    if (!err.status || err.status !== 400) {
```

Comment:
Is this not overly eager?

Reply:
how do you mean? 🤔

Reply:
not sure if that's what you meant but, I figured the `!err.status || ` part is redundant, so I've removed it now. In my head, when I initially added it, I wanted to catch cases where `status` was never even specified to begin with.

But please let me know if you were referring to something else.

Reply:
I was thinking we only want to silence if an attacker's URL couldn't be parsed. Any other error may be very relevant to see. So i was expecting a match on the URL exception i think ?

Reply:
I could indeed make the except that specific. But my angle was that, since the error is labelled with status `400`, it already indicates that it is an issue from the client, and no action/fix is necessarily required on the server. I also leaned more towards this because of what was mentioned about unnecessary logs that may increase log bills.

Reply:
If that assumption is :100: correct, i'm ok with this one. If there are doubts, i'd rather hide errors on a case by case basis so we're :100: sure we don't miss out on important ones. Right now, it's just the parsing of intentionally misformed URLs that is causing us problems, so maybe we should just hide that one

Reply:
Since I can't vouch for how diligently the `400` error status is adhered to, I updated the PR so it is more specific. When a URIError with a status `400` is found, the `err.message` is logged. This still allows to inspect the logs for the error message, but it excludes the err.name which may cause unnecessary alerts.